.devcontainer/cuda11.8-pip/devcontainer.json

```diff
@@ -10,8 +10,8 @@
   },
   "hostRequirements": {"gpu": "optional"},
   "features": {
-    "ghcr.io/rapidsai/devcontainers/features/ucx:24.2": {"version": "1.14.1"},
-    "ghcr.io/rapidsai/devcontainers/features/rapids-build-utils:24.2": {}
+    "ghcr.io/rapidsai/devcontainers/features/ucx:24.4": {"version": "1.14.1"},
```

Comment:
We updated all of RAPIDS 24.04 to use ucx 1.15, so this is out of date. This isn't a blocker for the release because the same issue exists in other RAPIDS repos (I checked RAFT). However, we should make a pass in 24.06 to fix these to 1.15.0. I wasn't aware that devcontainers hard-coded this. @trxcllnt could you help with updating this?

https://github.com/rapidsai/build-planning/issues/27#issuecomment-1997868433
https://github.com/rapidsai/integration/pull/704
https://github.com/rapidsai/ucx-py/pull/1032
https://github.com/rapidsai/ucxx/pull/205

Reply:
@bdice does this need to be done in this PR?

Reply:
> This isn't a blocker for the release because the same issue exists in other RAPIDS repos (I checked RAFT). However, we should make a pass in 24.06 to fix these to 1.15.0.

No, please target 24.06 for the update.